hubgrep/frontend_blueprint/templates/components/search_form/search_form.html

```diff
@@ -16,7 +16,7 @@
 
         {% import "components/collapsible/collapsible.html" as collapsible %}
         {% set collapsible_content -%}
-        <div class="group">
+        <div class="filter-section">
```

Comment:
couldnt help myself, made it more clear what it was (it wasnt used by css before, so no further refactoring needed)